slack.json

```diff
@@ -4951,12 +4951,12 @@
             },
             {
                 "module": "slack_sdk",
-                "input_file": "wheels/shared/slack_sdk-3.31.0-py2.py3-none-any.whl"
+                "input_file": "wheels/shared/slack_sdk-3.33.5-py2.py3-none-any.whl"
```

Comment:
Looks like you haven't removed the old wheels

Reply:
In the 'file changes', it shows that the **file has been renamed**. 
Thus, the old wheels have been removed.